server.js

```diff
@@ -1,38 +1,93 @@
 const express = require("express");
+const { sequelize } = require("./sequelize");
+
 const app = express();
 const port = 3000;
 
 app.use(express.json());
 
+async function fullBD() {
+  fetch(`https://jsonplaceholder.typicode.com/todos`)
+    .then(response => response.json())
+    .then(response =>{
+      response.forEach(async element => {
+        const { title, completed, id } = element;
+        const findtodo = await sequelize.findOne({ where: { id } });
+        if(findtodo){
+        await sequelize.create({
+          title,
+          completed,
+        });
+      }
+    });
+    })};
```

Comment:
why this new function? I think we don't need this, now we're working only with DB, no need to use the fake api anymore